packages/rancher-cis-benchmark/charts/templates/cis-roles.yaml

```diff
@@ -19,26 +16,6 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  labels:
-    rbac.authorization.k8s.io/aggregate-to-edit: "true"
-  namespace: {{ template "cis.namespace" . }}
-  name: cis-edit
```

Comment:
Are we getting rid of edit because it's the same as admin ? We'll need to update the docs.

Reply:
I think you suggested that in the design meeting? hence I removed it

Reply:
Ha, ok. Should be fine, only downside is if people already were using it but we can document that it changed to only admin.

Reply:
@cbron  added this docs PR as well: https://github.com/rancher/docs/pull/2785